packages/vue/src/composables/__tests__/useAuthenticator.spec.ts

```diff
@@ -59,7 +60,7 @@ describe('useAuthenticator', () => {
       state: mockState.value,
     });
 
-    expect(wrapper.vm.authStatus).toBe('authenticated');
```

Comment:
Why did this test previously assume you were authenticated?

Reply:
Oh, if you look at line 34 above i changed the mock value to `unauthenticated`. That probably was misleading... :D